src/modules/status.js

```diff
@@ -7,24 +7,23 @@ export default function status() {
   const hide = document.querySelectorAll('.hide');
   const icon = document.querySelectorAll('.jvicon');
   let value = false;
-  setTimeout(() => {
-    if (this.checked === true) {
-      value = true;
-      tasks[index].completed = value;
-      input[index].style.textDecoration = 'line-through';
-      input[index].style.color = 'rgba(139, 134, 134, 0.8)';
-      input[index].disabled = true;
-      icon[index].style.display = 'none';
-      hide[index].style.display = 'block';
-      localStorage.setItem('array', JSON.stringify(tasks));
-    } else {
-      tasks[index].completed = value;
-      input[index].style.textDecoration = 'none';
-      input[index].style.color = 'inherit';
-      input[index].disabled = false;
-      icon[index].style.display = 'block';
-      hide[index].style.display = 'none';
-      localStorage.setItem('array', JSON.stringify(tasks));
-    }
-  }, 100);
-}
\ No newline at end of file
+  // setTimeout(() => {
+  if (this.checked === true) {
+    value = true;
+    tasks[index].completed = value;
+    input[index].style.textDecoration = 'line-through';
+    input[index].style.color = 'rgba(139, 134, 134, 0.8)';
+    input[index].disabled = true;
+    icon[index].style.display = 'none';
+    hide[index].style.display = 'block';
+    localStorage.setItem('array', JSON.stringify(tasks));
+  } else {
+    tasks[index].completed = value;
+    input[index].style.textDecoration = 'none';
+    input[index].style.color = 'inherit';
+    input[index].disabled = false;
+    icon[index].style.display = 'block';
+    hide[index].style.display = 'none';
+    localStorage.setItem('array', JSON.stringify(tasks));
+  }
+}// , 100);
```

Comment:
- [ ] Please don't forget to remove old pieces of commented code, they make your code base look ugly and make it less readable. 🤓 🔧